telegram/ext/choseninlineresulthandler.py

```diff
@@ -86,20 +98,3 @@ def check_update(self, update):
 
         """
         return isinstance(update, Update) and update.chosen_inline_result
-
-    def handle_update(self, update, dispatcher):
-        """Send the update to the :attr:`callback`.
-
-        Args:
-            update (:class:`telegram.Update`): Incoming telegram update.
-            dispatcher (:class:`telegram.ext.Dispatcher`): Dispatcher that originated the Update.
-
-        """
-        optional_args = self.collect_optional_args(dispatcher, update)
-
-        return self.callback(dispatcher.bot, update, **optional_args)
-
-    # old non-PEP8 Handler methods
-    m = "telegram.ChosenInlineResultHandler."
-    checkUpdate = deprecate(check_update, m + "checkUpdate", m + "check_update")
-    handleUpdate = deprecate(handle_update, m + "handleUpdate", m + "handle_update")
```

Comment:
Please update `CHANGES.rst` and specify that these methods were removed.
It would be unpleasant if we'd forget doing so.

Reply:
They should've been removed ages ago... the other ones were... though I cannot figure out when...

Reply:
I think that we delayed all deprecations regarding Handlers until this refactoring effort.

Reply:
But... all the other deprecate statements are gone already....

Reply:
What does it matter? The code is there? We remove it **now**? We need to make sure its in the release notes. Even if we didn't intend to have the code there.

Reply:
But the original removal isn't even in the release notes... but alright, it's there now.